src/components/Molecules/Card/Card.js

```diff
@@ -1,27 +1,64 @@
 import React from 'react';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import PropTypes from 'prop-types';
 import Picture from '../../Atoms/Picture/Picture';
 import spacing from '../../../theme/shared/spacing';
 
 const Container = styled.div`
-  display: flex;
   position: relative;
-  flex-direction: column;
+  display: flex;
+  /* Check for smallBreakpointLayout prop coming from the CMS, otherwise column view default */
+  flex-direction: ${props => (props.smallBreakpointLayout === 'Row' ? 'row' : 'column')};
+  /* Check for mediumBreakpointLayout prop coming from the CMS */
+  @media ${({ theme }) => theme.breakpoint('small')} {
+    ${({ mediumBreakpointLayout }) => (mediumBreakpointLayout === 'Row')
+      && css`
+        flex-direction: row;
+      `}
+  }
+  /* Return all settings to column view for larger viewports */
+  @media ${({ theme }) => theme.breakpoint('medium')} {
+    flex-direction: column;
+  }
   height: 100%;
   border-radius: ${props => (props.squaredCorners ? '0' : '1rem')};
   overflow: hidden;
   background: ${({ theme, backgroundColor }) => theme.color(backgroundColor)};
 `;
 
-const Image = styled.div`
+const Wrapper = styled.div`
   height: auto;
 `;
 
 const Copy = styled.div`
-  padding: ${spacing('l')};
   display: flex;
   flex-direction: column;
+  padding: ${spacing('l')};
+  /* Check for smallBreakpointLayout prop coming from the CMS, otherwise column view default */
+  ${({ smallBreakpointLayout }) => (smallBreakpointLayout === 'Row')
+    && css`
+      padding: ${spacing('sm')};
+      h1, h2, h3, h4, h5 {
+       margin-bottom: 0.5rem;
```

Comment:
Would be cool to use the `spacing.js` variables whenever possible rather than hardcoding here